.github/workflows/github-merit-badger.yml

```diff
@@ -0,0 +1,20 @@
+name: github-merit-badger
+on:
+  pull_request_target:
+    types:
+     - opened
+
+jobs:
+  call-action:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: aws-github-ops/github-merit-badger@v0.0.98
+        id: merit-badger
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          badges: '[first-time-contributor,repeat-contributor,valued-contributor,seasoned-contributor,all-star-contributor,distinguished-contributor]'
```

Comment:
I'm not a fan of this. I get the idea of game theory as a motivator but I think we should stick with a level playing field and avoid this hierarchical social class system. It's just not a good look and can easily turn folks away.